t/03-testapi.t

```diff
@@ -9,11 +9,14 @@ use Test::Mock::Time;
 use File::Temp;
 use Test::Output qw(combined_like stderr_like);
 use Test::Fatal;
+use Test::MockObject;
+use Test::Exception;
 use Test::Warnings qw(:all :report_warnings);
 use Scalar::Util 'looks_like_number';
 
 use OpenQA::Isotovideo::Interface;
 use consoles::console;
+use backend::baseclass;
```

Comment:
As this test module t/03-testapi.t is already a bit big I suggest to move out this use and all corresponding tests into a different or its own test module so that we try to keep the "high level of test API only" in this test module

Reply:
Ok, I'm going for it.

Reply:
What is the trick, that the test is automatically loaded/run? I remember that I previously added it in some `Makefile`.

Reply:
And `done` from my side.

Reply:
It's not necessary anymore to explicitly add test files